main/options.c

```diff
@@ -3519,13 +3520,11 @@ struct preloadPathElt {
 
 static char* prependEnvvar (const char *path, const char* envvar)
 {
-	char *full_path = NULL;
-
 	const char* const envval = getenv (envvar);
-	if (envval)
-		full_path = combinePathAndFile(envval, path);
+	if (!envval)
+		return NULL;
 
-	return full_path;
+	return combinePathAndFile(envval, path);
```

Comment:
I believe we should not fallback to `/ctags` when the environment variable is not set.

Reply:
Good catch!

I would like to introduce a fix for this important suggestion first.
I made a pull request for the fix. See  #2385.